src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  padding: 100px;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    width: $width-blocks-calendar;
+    gap: $shift;
+    padding: $isnside-shift;
+
+    @for $index from 1 through $max-days-of-month {
+      & div:nth-child(#{$index}) {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: $width-blocks-day;
+        height: $width-blocks-day;
+        background-color: #eee;
+        border: 1px solid black;
+        transition-duration: 0.5s;
+        cursor: pointer;
```

Comment:
Don't put these and hover styles into the loop, these styles are common.

Use loop only to add numbers to the boxes.